internal/installcfg/installcfg.go

```diff
@@ -150,61 +149,39 @@ func getBasicInstallConfig(cluster *common.Cluster) *InstallerConfigBaremetal {
 	return cfg
 }
 
-// [TODO] - remove once we decide to use specific values from the hosts of the cluster
-func getDummyMAC(log logrus.FieldLogger, dummyMAC string, count int) (string, error) {
-	hwMac, err := net.ParseMAC(dummyMAC)
-	if err != nil {
-		log.Warn("Failed to parse dummyMac")
-		return "", err
-	}
-	hwMac[len(hwMac)-1] = hwMac[len(hwMac)-1] + byte(count)
-	return hwMac.String(), nil
-}
-
 func setBMPlatformInstallconfig(log logrus.FieldLogger, cluster *common.Cluster, cfg *InstallerConfigBaremetal) error {
 	// set hosts
 	numMasters := countHostsByRole(cluster, models.HostRoleMaster)
 	numWorkers := countHostsByRole(cluster, models.HostRoleWorker)
-	masterCount := 0
-	workerCount := 0
 	hosts := make([]host, numWorkers+numMasters)
 
-	// dummy MAC and port, once we start using real BMH, those values should be set from cluster
-	dummyMAC := "00:aa:39:b3:51:10"
-	dummyPort := 6230
-
-	for i := range hosts {
-		log.Infof("Setting master, host %d, master count %d", i, masterCount)
-		if i >= numMasters {
-			hosts[i].Name = fmt.Sprintf("openshift-worker-%d", workerCount)
-			hosts[i].Role = string(models.HostRoleWorker)
-			workerCount += 1
-		} else {
-			hosts[i].Name = fmt.Sprintf("openshift-master-%d", masterCount)
-			hosts[i].Role = string(models.HostRoleMaster)
-			masterCount += 1
-		}
-		hosts[i].Bmc = bmc{
-			Address:  fmt.Sprintf("ipmi://192.168.111.1:%d", dummyPort+i),
-			Username: "admin",
-			Password: "rackattack",
+	yamlHostIdx := 0
```

Comment:
why this var is needed? range is not returning index?

Reply:
we are skipping over disabled

Reply:
i see it thanks